docs/api/runtime.messages.md

```diff
@@ -48,7 +48,7 @@ const msgSet = {
     e: 'Minä puhun vain suomea.'
   }
 };
-writeFileSync('messages.js', String(mf.compile(msgSet)));
+writeFileSync('messages.js', String(mf.compile(JSON.stringify(msgSet))));

```

Comment:
This is not the right fix, for two reasons:
1. These API docs are generated by api-extractor/api-documenter from source code comments, so this change needs to be applied to `packages/runtime/src/messages.ts`.
2. This is a remnant of the old API, where the `compile()` method accepted object input. That functionality has been moved to `@messageformat/core/compile-module`, which ought to be used here instead.